implementations/rust/ockam/ockam_command/src/logs/mod.rs

```diff
@@ -64,14 +64,13 @@ pub fn setup_logging(
         }
         // If a log path is provided, log to a rolling file appender.
         Some(log_path) => {
-            let r = RollingFileAppender::new(
-                log_path,
-                RollingConditionBasic::new()
-                    .daily()
-                    .max_size(log_max_size_bytes()),
-                log_max_files(),
-            )
-            .expect("Failed to create rolling file appender");
+            let log_dir = log_path.parent().expect("Failed to get log directory");
+            let log_name = log_path
+                .file_name()
+                .expect("Failed to get log file name")
+                .to_str()
+                .unwrap();
+            let r = RollingFileAppender::new(Rotation::MINUTELY, log_dir, log_name);
```

Comment:
I wonder if `MINUTELY` is a bit too fined-grained, and if that will force us to jump too much between files when debugging. Let's see.

Reply:
oops I changed that while testing some stuff. It should be daily.